From: Jean Delvare <jdelvare@suse.de>
Subject: acer-wmi: Stop selecting VIDEO_OUTPUT_CONTROL
Patch-mainline: Submitted, 2014-03-17 - platform-driver-x86
Git-commit: 1c71a1b8a738621fd86cd787ea5887bf817d7022
Git-repo: https://git.kernel.org/cgit/linux/kernel/git/rafael/linux-pm.git/commit/?h=linux-next

ACPI_VIDEO no longer depends on VIDEO_OUTPUT_CONTROL, so drivers which
want to select ACPI_VIDEO no longer have to select
VIDEO_OUTPUT_CONTROL.

Signed-off-by: Jean Delvare <jdelvare@suse.de>
Cc: "Lee, Chun-Yi" <jlee@suse.com>
Cc: Matthew Garrett <matthew.garrett@nebula.com>
---
 drivers/platform/x86/Kconfig |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/platform/x86/Kconfig
+++ b/drivers/platform/x86/Kconfig
@@ -27,8 +27,6 @@ config ACER_WMI
 	depends on ACPI_WMI
 	select INPUT_SPARSEKMAP
 	# Acer WMI depends on ACPI_VIDEO when ACPI is enabled
-	# but for select to work, need to select ACPI_VIDEO's dependencies, ick
-        select VIDEO_OUTPUT_CONTROL if ACPI
         select ACPI_VIDEO if ACPI
 	---help---
 	  This is a driver for newer Acer (and Wistron) laptops. It adds
